modules/solid_properties/include/userobjects/ThermalUCProperties.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
```

Comment:
add moose copyright header

same for other files

Reply:
also solid properties are now in the solidproperties folder no longer in userobjects/

use `git mv` to move them pls